common/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1032,6 +1032,7 @@ dao.compensation.display.link=Link to detail info:
 dao.compensation.display.link.prompt=Provide a link to an issue at \"https://github.com/bisq-network/compensation/issues/\" or the Bisq Forum.
 dao.compensation.display.requestedBsq=Requested amount in BSQ:
 dao.compensation.display.bsqAddress=BSQ address:
+dao.compensation.display.txId=BTC TxID:
```

Comment:
I would prefer: "BTC transaction ID:" if space in UI allows, otherwise tx ID

Reply:
Fixed.